README.rst

```diff
@@ -36,15 +36,17 @@ Example:
         username: deedee
         email: deedee@example.com
 
-    Profile:
+    - Profile:
       - user: dee
         name: Douglas
 
-    Group:
+    - Group:
       - name: Ramones
         members: [joey.profile, dee.profile]
 
--  The root of YAML contain ``mapper`` names i.e. ``User``
+-  The root of YAML contain ``mapper`` names e.g. ``User``, ``Profile`` etc
```

Comment:
it is not a mapper. it is a list (as you said below). where an element is a mapping with a single element.